src/helpers/file.test.ts

```diff
@@ -323,4 +325,78 @@ describe('File helpers', () => {
       });
     });
   });
+
+  describe('isExisting', () => {
+    describe('given target file', () => {
+      it('returns true if the target file exists', () => {
+        const target = 'targetFile.txt';
+        const projectName = 'projectName';
+        const targetPath = getProjectFilePath(target, projectName);
+        const existSpy = jest.spyOn(fs, 'existsSync');
+
+        (fs.existsSync as jest.Mock).mockReturnValue(true);
+
+        expect(isExisting(target, projectName)).toBe(true);
+        expect(existSpy).toHaveBeenCalledWith(targetPath);
+      });
+
+      it('returns false if the target file does not exist', () => {
+        const target = 'targetFile.txt';
+        const projectName = 'projectName';
+        const targetPath = getProjectFilePath(target, projectName);
+        const existSpy = jest.spyOn(fs, 'existsSync');
+
+        (fs.existsSync as jest.Mock).mockReturnValue(false);
+
+        expect(isExisting(target, projectName)).toBe(false);
+        expect(existSpy).toHaveBeenCalledWith(targetPath);
+      });
+    });
+  });
+
+  describe('containsContent', () => {
+    describe('given target file and content', () => {
+      it('returns true if the target file contains the content', () => {
```

Comment:
```suggestion
    describe('given a target file that contains content', () => {
      it('returns true', () => {
```

Should we move the condition in the `describe` block? Then we can have a second describe block for the next test like:

````
    describe('given a target file that does NOT contain content', () => {
      it('returns false', () => {
````

Reply:
If you agree with this suggestion, please also change for the `isExisting` tests 🙇

Reply:
Nice, I improved all of them in https://github.com/nimblehq/infrastructure-templates/pull/169/commits/82dcb1d3ece96fa28454a09887aa393d007397a2